src/index.html

```diff
@@ -11,12 +11,20 @@
 <body>
     <div id="app">
         <header>
-            <div id="display">
-                <p></p>
-            </div>
+            <div id="display"></div>
+            <p id="add-number"></p>
         </header>
 
-        <main></main>
+        <main>
+        </main>
+        <div id="call-delete-wrapper">
+            <button id="call-button" class="dial-button">
+                <h2>Call</h2>
```

Comment:
Same here, h2 inside button is not great idea - I would use p and style it according to your needs.